commit 26f274011096b9eacfd2753d8f7afcb63e796ab9
Author: nicm <nicm>
Date:   Thu Aug 1 11:45:34 2019 +0000

    xterm 348 now disables margins when resized, so send DECLRMM again.

diff --git a/tty.c b/tty.c
index ab1da9fd39..024aef2712 100644
--- a/tty.c
+++ b/tty.c
@@ -1879,6 +1879,8 @@ tty_invalidate(struct tty *tty)
 	tty->rlower = tty->rright = UINT_MAX;
 
 	if (tty->flags & TTY_STARTED) {
+		if (tty_use_margin(tty))
+			tty_puts(tty, "\033[?69h"); /* DECLRMM */
 		tty_putcode(tty, TTYC_SGR0);
 
 		tty->mode = ALL_MODES;
